packages/pnpm-sync-lib/etc/pnpm-sync-lib.api.md

```diff
@@ -4,6 +4,9 @@
 
 ```ts
 
+// @beta
+export function getPnpmSyncJsonVersion(): string;

```

Comment:
With this change, the 3 exported functions are now:

- `getPnpmSyncJsonVersion()`
- `pnpmSyncCopyAsync()`
- `pnpmSyncPrepareAsync()`

The common coding convention is for function names to start with a verb.  Therefore I understood `pnpmSync` in `pnpmSyncCopyAsync()` to be a namespace prefix, such that the API consumer would do this:
```ts
import { pnpmSyncCopyAsync } from `pnpm-sync`

// (all the pnpm-sync APIs start with "pnpmSync")
```
instead of a namespace import:
```ts
import * as pnpmSync from `pnpm-sync`

await pnpmSync.copyAsync()

// obviously pnpmSync.pnpmSyncCopyAsync() would be needlessly verbose
```

Under that theory, `getPnpmSyncJsonVersion()` should be called `pnpmSyncGetJsonVersion()`.

But it is odd that there are a bunch of other types, e.g. your Rush implementation is doing this:
```ts
import {
  type ILockfile,
  pnpmSyncPrepareAsync,
  type ILogMessageCallbackOptions,
  type ILockfilePackage
} from 'pnpm-sync-lib';
```

🤔

It's tempting to introduce a proper scope for these functions:

```ts
import {
  type ILockfile,
  pnpmSync,
  type ILogMessageCallbackOptions,
  type ILockfilePackage
} from 'pnpm-sync-lib';

await pnpmSync.prepareAsync()
```
...or else standardize on the namespace imports.

But maybe it's not worth changing. 🙂

Reply:
I will think about it in about PR regarding the standardization on the namespace imports.

In this PR, let me rename the function name first.